packages/suite/src/components/suite/modals/AddAccount/components/EnableNetwork.tsx

```diff
@@ -71,44 +55,28 @@ export const EnableNetwork = ({
                 networks={networks}
                 selectedNetworks={selectedNetworks}
             />
+
             {hasTestnetNetworks && (
-                <>
-                    <TestnetCoinsTrigger
-                        onClick={() => {
-                            setTestnetVisible(!isTestnetVisible);
-                        }}
-                    >
-                        <Label isTestnetVisible={isTestnetVisible}>
+                <StyledCollapsibleBox
+                    variant="small"
+                    heading={
+                        <P size="tiny" weight="bold">
                             <Translation id="TR_TESTNET_COINS" />
-                        </Label>
-                        <Icon
-                            canAnimate
-                            isActive={isTestnetVisible}
-                            size={16}
-                            color={theme.TYPE_LIGHT_GREY}
-                            icon="ARROW_DOWN"
+                        </P>
+                    }
+                >
+                    <TestnetCoinsWrapper>
+                        <TestnetCoinsDescription>
+                            <StyledIcon size={12} color={theme.TYPE_LIGHT_GREY} icon="INFO" />
```

Comment:
![Screenshot 2022-09-05 at 10 09 50](https://user-images.githubusercontent.com/33235762/188400589-a72af130-2dc9-46c9-9b16-0758faa936fb.png)
The icon is so small that it does not look like a question mark

Reply:
Btw would be nice to have this info also in Settings > Crypto

Reply:
donedad90c9f640f4efd44175ba250ecfb8088ab2a92

Reply:
It is now in Settings > Crypto. However, the icon on the non-retina display is still blurred